src/Components/Content.tsx

```diff
@@ -1,238 +1,211 @@
-    color: #e7e7eb;
-    `;
-
-    const AirPressureData = styled.div`
-    font-weight: 700;
-    font-size: 45px;
-    font-family: Raleway,sans-serif;
-    color: #e7e7eb;
-    padding-bottom: 20px;
-    `;
-
-    const AirPressureDesc = styled.span`
-    font-size: 25px;
-    font-weight: 400;
-    font-family: Raleway,sans-serif;
-    color: #e7e7eb;
-    `;
-
-    const TopBtns = styled.div`
-    margin-top: 40px;
-    text-align: right;
-    margin-left: 87%;
-    display: flex;
-    `;
-
-    const Btn = styled.div`
-    border-radius: 100%;
-    padding: 15px;
-    font-family: Raleway,sans-serif;
-    font-style: normal;
-    font-weight: 700;
-    font-size: 14px;
-    margin-right: 38px;
-    color: #110e3c;
-    text-align: center;
-    background: #585676;
-    cursor: pointer;
-    `;
-
-    const ProgresBar = styled.div`
-    width: 80%;
-    height: 7px;
-    background: #fff;
-    border-radius: 50px;
-    overflow: hidden;
-    margin: auto;
-    margin-bottom: 20px;
-    `;
-
-    const BarFill = styled.div`
-    height: 8px;
-    background: #ffec65;
-    width: 81%;
-    `;
-
-return (
-    <Container>
-        <TopBtns>
-            <Btn>°C</Btn>
-            <Btn>°F</Btn>
-        </TopBtns>
-
-        <HightlightsTitle>Today's Hightlights </HightlightsTitle>
-        <HightlightsDetails>
-            <WindStatus>
-                <WindStatusTitel> Wind status </WindStatusTitel>
-                <WindData>{todaysForecast?.hour[0].wind_mph}<WindDesc>mph</WindDesc></WindData>
-            </WindStatus>
-
-            <Humidity>
-                <HumidityTitel>Humidity</HumidityTitel>
-                <HumidityData>{todaysForecast?.hour[0].humidity}<HumidityDesc>%</HumidityDesc>
-                <ProgresBar>
-                <BarFill data-v-042b7e1e=""></BarFill>
-                </ProgresBar>
-                </HumidityData>
-            </Humidity>
-
-            <Visibility>
-                <VisibilityTitel>Visibility</VisibilityTitel> 
-                <VisibilityData>{todaysForecast?.hour[0].vis_miles}<VisibilityDesc>miles</VisibilityDesc>
-                </VisibilityData>
-                </Visibility>
-
-            <AirPressure>
-                <AirPressureTitel>Air Pressure </AirPressureTitel>
-                <AirPressureData>{todaysForecast?.hour[0].pressure_mb}<AirPressureDesc>hPa</AirPressureDesc>
-                </AirPressureData>
-                </AirPressure>
-        </HightlightsDetails>
-    </Container>
-    )
+    todaysForecast?: ForecastEachDay;
+    setIsTempC: (isTempC: boolean) => void;
+    isTempC: boolean;
+}
+
+const Container = styled.div`
+position: absolute;
+width: 70%;
+right: 0;
+top: 0;
+background: #100e1d;
+min-height: 100vh;
+color: white;
+`;
+
+const TopBtns = styled.div`
+margin-top: 22px;
+text-align: right;
+margin-left: 87%;
+display: flex;
+`;
+
+const Btn = styled.div`
+border-radius: 100%;
+padding: 15px;
+font-family: Raleway,sans-serif;
```

Comment:
Try to avoid using the same CSS rules for all components. You can still have some "global" styles with `StyledComponents` ;)